WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StatsTimeRangeBarViewModel.swift

```diff
@@ -83,53 +83,39 @@ struct StatsTimeRangeBarViewModel: Equatable {
                                                 endDate: endDate,
                                                 timezone: timezone)
     }
-
-    init(startDate: Date,
-         endDate: Date,
-         selectedDate: Date,
-         timeRange: StatsTimeRangeV4,
-         timezone: TimeZone) {
-        // Disable editing time range when selecting a specific date on the graph
-        isTimeRangeEditable = false
-        granularityText = nil
-        timeRangeText = timeRange.timeRangeText(startDate: startDate,
-                                                endDate: endDate,
-                                                selectedDate: selectedDate,
-                                                timezone: timezone)
-    }
```

Comment:
We should keep this initializer because we want to display the selected time when tapping on any time range on the chart.